test/mpi/pt2pt/testlist.in

```diff
@@ -15,6 +15,8 @@ bsendpending 2
 isendself 1
 issendselfcancel 1 xfail=ticket2276
 isendirecv 10
+isendrecv 10
```

Comment:
Could we lower the # of processes to something like 4 or 8? I'm not sure 10 had any impact on the outcome of the test, and it might overrun the resources of some smaller machines (laptops).

Reply:
Lowered to 4.